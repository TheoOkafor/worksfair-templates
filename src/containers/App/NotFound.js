import React, { Fragment } from 'react';
import { Image, Transformation } from 'cloudinary-react';
import { Helmet } from 'react-helmet';

export default () => {
  const compass =
    'https://res.cloudinary.com/worksfair/image/upload/v1564834329/dev/qvwvfuxg6040ppipkanj.png';
  return (
    <Fragment>
      <Helmet>
        <title>Not Found - Worksfair</title>
      </Helmet>
      <div
        className="hero is-large has-background-light"
      >
        <div
          className="hero-body"
          style={{ padding: '50px' }}
        >
          <div className="container columns">
            <div className="column is-1" />
            <div className="column is-7">
              <h1
                style={{ fontSize: '70px', marginTop: '20px' }}
                className="has-text-extra-large"
              >
                404
              </h1>
              <p
                style={{ fontSize: '30px', marginTop: '15px' }}
                className="has-text-large"
              >
                We are sorry about this.
              </p>
              <p
                style={{ fontSize: '20px', marginTop: '15px' }}
                className="is-size-5"
              >
                The website you are looking for is not here.
              </p>
              <p
                className="margin-top-25"
                style={{ marginTop: '20px' }}
              >
                <a
                  href="http://www.worksfair.com"
                  className="button is-info is-medium"
                >
                  Go to worksfair
                </a>
              </p>
            </div>
            <div className="column is-3 has-text-extra-large">
              <Image
                cloudName="worksfair"
                publicId={compass}
                type="fetch"
                className="work-image"
              >
                <Transformation width="700" fetchFormat="auto" />
              </Image>
            </div>
          </div>
        </div>
      </div>
    </Fragment>
  );
};
