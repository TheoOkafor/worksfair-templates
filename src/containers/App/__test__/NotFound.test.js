import React from 'react';
import {cleanup, render} from '@testing-library/react';

import NotFound from '../NotFound';

afterEach(cleanup);

describe('NotFound.js', () => {
  test('test that it renders correctly', () => {
    const component = render(
      <NotFound />
    );
    expect(component).toMatchSnapshot();
  });
});
