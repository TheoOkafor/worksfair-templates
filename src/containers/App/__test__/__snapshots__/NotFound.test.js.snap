// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.js test that it renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="hero is-large has-background-light"
      >
        <div
          class="hero-body"
          style="padding: 50px;"
        >
          <div
            class="container columns"
          >
            <div
              class="column is-1"
            />
            <div
              class="column is-7"
            >
              <h1
                class="has-text-extra-large"
                style="font-size: 70px; margin-top: 20px;"
              >
                404
              </h1>
              <p
                class="has-text-large"
                style="font-size: 30px; margin-top: 15px;"
              >
                We are sorry about this.
              </p>
              <p
                class="is-size-5"
                style="font-size: 20px; margin-top: 15px;"
              >
                The website you are looking for is not here.
              </p>
              <p
                class="margin-top-25"
                style="margin-top: 20px;"
              >
                <a
                  class="button is-info is-medium"
                  href="http://www.worksfair.com"
                >
                  Go to worksfair
                </a>
              </p>
            </div>
            <div
              class="column is-3 has-text-extra-large"
            >
              <img
                class="work-image"
                src="http://res.cloudinary.com/worksfair/image/fetch/f_auto/https://res.cloudinary.com/worksfair/image/upload/v1564834329/dev/qvwvfuxg6040ppipkanj.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="hero is-large has-background-light"
    >
      <div
        class="hero-body"
        style="padding: 50px;"
      >
        <div
          class="container columns"
        >
          <div
            class="column is-1"
          />
          <div
            class="column is-7"
          >
            <h1
              class="has-text-extra-large"
              style="font-size: 70px; margin-top: 20px;"
            >
              404
            </h1>
            <p
              class="has-text-large"
              style="font-size: 30px; margin-top: 15px;"
            >
              We are sorry about this.
            </p>
            <p
              class="is-size-5"
              style="font-size: 20px; margin-top: 15px;"
            >
              The website you are looking for is not here.
            </p>
            <p
              class="margin-top-25"
              style="margin-top: 20px;"
            >
              <a
                class="button is-info is-medium"
                href="http://www.worksfair.com"
              >
                Go to worksfair
              </a>
            </p>
          </div>
          <div
            class="column is-3 has-text-extra-large"
          >
            <img
              class="work-image"
              src="http://res.cloudinary.com/worksfair/image/fetch/f_auto/https://res.cloudinary.com/worksfair/image/upload/v1564834329/dev/qvwvfuxg6040ppipkanj.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
